File: Boare.Lib.AppUtil/MessageBodyEntry.cs
/*
 * MessageBody.cs
 * Copyright (c) 2009 kbinani
 *
 * This file is part of Boare.Lib.AppUtil.
 *
 * Boare.Lib.AppUtil is free software; you can redistribute it and/or
 * modify it under the terms of the BSD License.
 *
 * Boare.Lib.AppUtil is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 */
#if JAVA
package org.kbinani.apputil;

import java.util.*;
#else
using System;
using bocoree.java.util;

namespace Boare.Lib.AppUtil {
#endif

    public class MessageBodyEntry {
        public String message;
        public Vector<String> location = new Vector<String>();

        public MessageBodyEntry( String message_, String[] location_ ) {
            message = message_;
            for ( int i = 0; i < location_.Length; i++ ) {
                location.add( location_[i] );
            }
        }
    }

#if !JAVA
}
#endif

File: port_java_OBSOLUTE/com/boare/util/MessageBodyEntry.java
/*
 * MessageBodyEntry.java
 * Copyright (c) 2009 kbinani
 *
 * This file is part of com.boare.util.
 *
 * com.boare.util is free software; you can redistribute it and/or
 * modify it under the terms of the BSD License.
 *
 * com.boare.util is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 */
package com.boare.util;

import java.util.*;

public class MessageBodyEntry {
    public String Message;
    public Vector<String> Location = new Vector<String>();

    public MessageBodyEntry( String message, String[] location ) {
        Message = message;
        for ( int i = 0; i < location.length; i++ ) {
            Location.add( location[i] );
        }
    }
}
